apps/api/src/deployment/services/deployment/deployment.service.ts

```diff
@@ -0,0 +1,83 @@
+import { MsgCreateDeployment } from "@akashnetwork/akash-api/v1beta3";
+import { SDL } from "@akashnetwork/akashjs/build/sdl";
+import { getAkashTypeRegistry } from "@akashnetwork/akashjs/build/stargate";
+import { DirectSecp256k1HdWallet, Registry } from "@cosmjs/proto-signing";
+import { SigningStargateClient } from "@cosmjs/stargate";
+import axios from "axios";
+import { singleton } from "tsyringe";
+
+import { BillingConfigService } from "@src/billing/services/billing-config/billing-config.service";
+import { SingleDeploymentResponse } from "@src/deployment/http-schemas/deployment.schema";
+import { MessageTransmitterService } from "@src/deployment/services/message-transmitter/message-transmitter.service";
+import { RestAkasheploymentInfoResponse, RestAkashLeaseListResponse } from "@src/types/rest";
+import { apiNodeUrl, betaTypeVersion, betaTypeVersionMarket } from "@src/utils/constants";
+import { env } from "@src/utils/env";
+
+@singleton()
+export class DeploymentService {
+  constructor(
+    private readonly config: BillingConfigService,
+    private readonly messageTransmitterService: MessageTransmitterService,
+  ) { }
+
+  public async findByOwnerAndDseq(owner: string, dseq: string): Promise<SingleDeploymentResponse['data']> {
+    const response = await axios.get<RestAkasheploymentInfoResponse>(`${apiNodeUrl}/akash/deployment/${betaTypeVersion}/deployments/info`, {
```

Comment:
This will be the perfect candidates for @stalniy's new chain-sdk implemention. Technically we [already have an endpoint implemented](https://github.com/akash-network/console/blob/f1a0c32422b3716010f936aa7d0a4abe38ee4762/apps/api/src/services/external/apiNodeService.ts#L361) for this one, but not refactored in the new services architecture. We should abstract those http calls in the [http package](https://github.com/akash-network/console/tree/f1a0c32422b3716010f936aa7d0a4abe38ee4762/packages/http-sdk).